src/python/tests/core/crash_analysis/stack_parsing/stack_analyzer_test.py

```diff
@@ -1591,7 +1591,7 @@ def test_syzkaller_kasan(self):
                       'sock_def_readable\n'
                       'unix_dgram_sendmsg\n')
     expected_address = '0xffffffc01640e9d0'
-    expected_stacktrace = data
+    expected_stacktrace = stack_analyzer.remove_kernel_times(data)
```

Comment:
We shouldnt call function here in test code, this should be an "kasan_syzkaller_stripped.txt" file and then we read it here for raw contents. if the files are same (no times), then use same file kasan_syzkaller.txt.